cherryontech/jupiter-jumpers
# Updates

## Description

1) Updates the favorites page so that the right bar collapses unless a log is clicked or it is manually opened.
2) Adds an updated default avatar
3) Updates so that when the user clicks "Calendar" the log summary list always renders in the right bar (no carry over from the favorites page).
4) Updates the favorites page so that if there aren't any favorited logs in the display, the right bar indicates that with a message.

## Testing Instructions

Play around with the favorites page: 
- [] Ensure that the right bar is collapsed when you click "Favorites"
- [] view it with zero favorited logs; ensure a message is shown when you open the right bar
- [] View it with one favorited log; open the right bar and ensure it shows that single log.
- [] Unfavorite that log, and ensure it disappears from the Favorites page, and that the right bar collapses. Re-open that bar and ensure the appropriate message is shown.
- [] Click back to 'Calendar' and ensure there is no carry-over in the right bar from the Favorites page. It should show the summary list. 

- [] Ensure the default avatar is shown
